src/template/Menu/Menu.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Icon, Menu } from 'antd'
+
+import logo from '../../assets/images/logo.png'
+import '../scss/Menu.scss'
+import { SITE_NAME } from '../../common/constants'
+// import Search from '../../../ui-kit/Search/Search'
```

Comment:
why commented code? do we need this ?

Reply:
I created the component but now I do not want it to be visible and in the future this component will be applied to the menu bar

Reply:
Cool, I get your point - but we NEVER tend to use commented code in our projects, it makes the code ugly and less understandable, we can always return to this Menu component at a later point and add the **Search** component